docs/topics_5/proc_web-troubleshooting-common-issues.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+[id="proc_web-troubleshooting-common-issues_{context}"]
+= Common issues when troubleshooting a {WebName} installation on OpenShift Container Platform
+
+.`NoRouteToHostException` error
+
+The following error might appear in the `mta-web-console-executor` pod log:
+
+[source,terminal]
+----
+13:44:03,501 SEVERE [org.jboss.windup.web.messaging.executor.ExecutorBootstrap] (main) Could not start messaging listener due to: Failed to connect to any server. Servers tried: [http-remoting://192.0.2.4:8080 (java.net.NoRouteToHostException: No route to host)]: javax.naming.CommunicationException: Failed to connect to any server. Servers tried: [http-remoting://192.0.2.4:8080 (java.net.NoRouteToHostException: No route to host)]
+----
+
+This error message is displayed in a new deployment because the `mta-web-console-executor` pod starts before the `mta-web-console` pod. Wait for the `mta-web-console` pod to start.
+
+.Insufficient resources
+
+The `mta-web-console` pod is not running and the following error is displayed on the *Events* tab of the *Pod Details* screen in the OpenShift console:
+
+[source,terminal]
+----
+0/9 nodes are available: 4 Insufficient cpu, 4 MatchNodeSelector, 9 Insufficient memory.
+----
+
+The resource quota for the project has been exceeded and the pod is unable to obtain the requested resources.
+
+You can either increase the link:{OpenShiftDocsURL}/applications/quotas/quotas-setting-per-project.html[resource quotas] and  link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-limit-ranges.html[limit ranges] or reduce the requested resources of the project to a minimum of 4 vCPUs and 8 GB.
+
+.Deployment pod timeout
+
+The `mta-web-console-deploy`, `mta-web-console-executor-deploy`, or `mta-web-console-postgresql-deploy` pod times out and the following error is displayed in its log:
+
+[source,terminal]
+----
+error: update acceptor rejected mta-web-console-executor-1: Pods for rc 'mta/<pod>' took longer than 600 seconds to become available
+----
+
+This error is probably caused by insufficient cluster resources.
+
+.Procedure
+
+. Redeploy the {WebName} and observe the `mta-web-console-deploy`, `mta-web-console-executor-deploy`, and `mta-web-console-postgresql-deploy` pod logs.
```

Comment:
This step is weird. If the user knows the cause from the pod logs, what is the point of redeploying the console and watching the logs do exactly the same thing? Remove this step.